tests/ballerina-test/src/test/java/org/ballerinalang/test/statements/foreach/ForeachJSONTests.java

```diff
@@ -97,7 +97,7 @@ public void testJSONNull() {
         Assert.assertEquals(returns[0].stringValue(), result);
     }
 
-    @Test
+    @Test (enabled = false)
```

Comment:
Why is this test disabled?

Reply:
json to strut casting was not working by the time of earlier commit. Now it's working fine. Enabled the test again.